src/localStorageHandlers.js

```diff
@@ -0,0 +1,29 @@
+export const setToLocal = (monster) => {
```

Comment:
Looks very clean and modular especially having these in a separate file and then importing them in when needed - nice!